meta-oe/recipes-core/sdbus-c++/sdbus-c++-tools-native_0.7.1.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "sdbus-c++ native tools"
+DESCRIPTION = "Native interface code generator for development with sdbus-c++"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git-tools/COPYING;md5=1803fa9c2c3ce8cb06b4861d75310742"
+
+inherit cmake native
+
+DEPENDS_${PN} = "expat"
+RDEPENDS_${PN} = "expat"
+
+BRANCH = "master"
+TAG = "v${PV}"
```

Comment:
Please use hardcoded git SHA instead of tags, tags are floating.

Reply:
Done